software/openvisualizer/openvisualizer/moteState/moteState.py

```diff
@@ -410,6 +410,34 @@ class moteState(eventBusClient.eventBusClient):
     ]
     
     TRIGGER_DAGROOT     = 'DAGroot'
+    SET_GD_IM1          = 'SetGoldenImage1'
```

Comment:
`SET_GD_IM1` and `SET_GD_IM2` are removed since they are not used. It's probably I want to make separate command for root and sniffer image. But I changed my mind and using one command but differentiate image with parameters.